[CICD] RedissonClient 빈 생성 실패

## 개요
- Redis Host인 playlist-redis에 연결 불가
- ECS Fargate 환경에서 playlist-app과 playlist-redis 컨테이너를 같은 태스크에 띄우고 있음
- task-definition.json과 GitHub Actions depoly.yml에서 환경변수 REDIS_HOST, REDIS_PORT, REDIS_PASSWORD를 지정함
- 그런데도 RateLimitService에서 의존하는 RedissonClient 빈 생성 실패

## 상세 설명
- 에러 로그
```
Caused by: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with
 name 'rateLimitService' defined in URL [jar:nested:/app/app.jar/!BOOT-INF/classes/!/com/codeit/playlist/domain/auth/passwordratelimit/RateLimitService.class]: 
Unsatisfied dependency expressed through constructor parameter 0: Error creating bean 
with name 'redissonClient' defined in class path resource [com/codeit/playlist/global/config/RateLimitConfig.class]: 
Failed to instantiate [org.redisson.api.RedissonClient]: Factory method 'redissonClient' threw exception 
with message: java.util.concurrent.ExecutionException: org.redisson.client.RedisConnectionException: 
java.net.UnknownHostException: Failed to resolve 'playlist-redis' [A(1)]
...
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean 
with name 'redissonClient' defined in class path resource [com/codeit/playlist/global/config/RateLimitConfig.class]: 
Failed to instantiate [org.redisson.api.RedissonClient]: Factory method 'redissonClient' threw exception 
with message: java.util.concurrent.ExecutionException: org.redisson.client.RedisConnectionException:
 java.net.UnknownHostException: Failed to resolve 'playlist-redis' [A(1)]
```

--- a/src/main/java/com/codeit/playlist/global/config/RateLimitConfig.java
+++ b/src/main/java/com/codeit/playlist/global/config/RateLimitConfig.java
@@ -9,25 +9,18 @@ import org.springframework.context.annotation.Configuration;
 
 @Configuration
 public class RateLimitConfig {
-
-  @Value("${spring.data.redis.host:localhost}")
-  private String redisHost;
-
-  @Value("${spring.data.redis.port:6379}")
-  private int redisPort;
-
-  @Value("${spring.data.redis.password:}")
-  private String redisPassword;
-
-  @Bean
-  public RedissonClient redissonClient() {
-    Config config = new Config();
-    var serverConfig = config.useSingleServer()
-        .setAddress(String.format("redis://%s:%d", redisHost, redisPort));
-    if (!redisPassword.isEmpty()) {
-      serverConfig.setPassword(redisPassword);
+    @Bean
+    public RedissonClient redissonClient(
+            @Value("${spring.data.redis.host:localhost}") String host,
+            @Value("${spring.data.redis.port:6379}") int port,
+            @Value("${spring.data.redis.password:}") String password) {
+        Config config = new Config();
+        var serverConfig = config.useSingleServer()
+                .setAddress(String.format("redis://%s:%d", host, port));
+        if (!password.isEmpty()) {
+            serverConfig.setPassword(password);
+        }
+        return Redisson.create(config);
     }
-    return Redisson.create(config);
-  }
 
 }

